### ISSUE TITLE
[mesheryctl] Activity Spinner: Grammar in `system stop`

### ISSUE THREAD

**Issue by leecalcote** [layer5io/meshery#3276]
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
When stopping Meshery, Meshery Operator and its custom controllers (pods) will be removed from the connected Kubernetes cluster. An activity spinner shows while the pods are being shutdown. Once pod termination is complete, a final message is shown, but conflicts with other text on the same line. See the last line in the example output.

#### Desired Behavior
Final output from the activity spinner should clear the entire line, not overlap with other output.

#### Screenshots/Logs
```
➜  ~ mesheryctl system stop
Stopping Meshery...
Stopping meshery_meshery-consul_1       ... done
Stopping meshery_meshery-cpx_1          ... done
Stopping meshery_meshery_1              ... done
Stopping meshery_meshery-osm_1          ... done
Stopping meshery_meshery-traefik-mesh_1 ... done
Stopping meshery_meshery-linkerd_1      ... done
Stopping meshery_watchtower_1           ... done
Stopping meshery_meshery-nsm_1          ... done
Stopping meshery_meshery-istio_1        ... done
Stopping meshery_meshery-kuma_1         ... done
Removing meshery_meshery-consul_1       ... done
Removing meshery_meshery-cpx_1          ... done
Removing meshery_meshery_1              ... done
Removing meshery_meshery-osm_1          ... done
Removing meshery_meshery-traefik-mesh_1 ... done
Removing meshery_meshery-linkerd_1      ... done
Removing meshery_watchtower_1           ... done
Removing meshery_meshery-nsm_1          ... done
Removing meshery_meshery-istio_1        ... done
Removing meshery_meshery-kuma_1         ... done
Pods terminatedeshery pods
Meshery is stopped.
```

#### Environment

- Host OS: Mac
- Meshery Client Version: stable-v0.5.26
- Platform: Docker

---

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)
- [Layer5 Community Slack](http://slack.layer5.io)

**Comment by leecalcote** [comment]
@NagaHarshita would you like to take this?

**Comment by NagaHarshita** [comment]
Yeah sure.

**Comment by DantrazTrev** [comment]
Hmm🤔 seems kinda related to #3255 . I believe that might help as starting point.

**Comment by leecalcote** [comment]
Oh, no! I might’ve opened up a duplicate issue. @DantrazTrev, will you evaluate And determine whether or not they are duplicative? If so, let’s close this one in favor of the original. Good eye.

**Comment by DantrazTrev** [comment]
> Oh, no! I might’ve opened up a duplicate issue. @DantrazTrev, will you evaluate And determine whether or not they are duplicative? If so, let’s close this one in favor of the original. Good eye. 

@leecalcote Oh yes on a closer look this is an duplicate issue. It seems to have been fixed by a PR already

**Comment by hexxdump** [comment]
Duplicate of #3255. Fix: #3276 is merged already.
